src/autotype/AutoType.cpp

```diff
@@ -199,36 +286,36 @@ void AutoType::performGlobalAutoType(const QList<Database*>& dbList)
 
     m_inAutoType = true;
 
-    QList<Entry*> entryList;
-    QHash<Entry*, QString> sequenceHash;
+    QList<AutoTypeMatch> matchList;
 
     for (Database* db : dbList) {
         const QList<Entry*> dbEntries = db->rootGroup()->entriesRecursive();
         for (Entry* entry : dbEntries) {
-            QString sequence = autoTypeSequence(entry, windowTitle);
-            if (!sequence.isEmpty()) {
-                entryList << entry;
-                sequenceHash.insert(entry, sequence);
+            const QList<QString> sequences = autoTypeSequences(entry, windowTitle);
+            for (QString sequence : sequences) {
```

Comment:
QString&

Reply:
It's a `QList<QString>`, adding `&` trigger this compiler error `error: binding ‘const QString’ to reference of type ‘QString&’ discards qualifiers`

Reply:
Yeah, well. QString const& it is then. ;-)